tests/generate_test.js

```diff
@@ -29,24 +29,18 @@ describe('generateCSSRuleset', () => {
         }, '.foo{background-color:red !important;}');
     });
 
-    it('prefixes vendor props with a dash', () => {
+    it('prefixes transition to webkit', () => {
```

Comment:
@xymostech `-ms-transition` actually never existed. `-moz-` was left out as it was only used up to Firefox v15 (~2012).